core/shared/src/main/scala/com/lynbrookrobotics/potassium/units/spacialValues.scala

```diff
@@ -7,6 +7,7 @@ class Point(override val x: Length,
             override val z: Length) extends Value3D(x, y, z) {
   def this(value3D: Value3D[Length]) = this(value3D.x, value3D.y, value3D.z)
   def this(x: Length, y: Length) = this(x, y, Feet(0))
+  def this() = this(Feet(0), Feet(0), Feet(0))
```

Comment:
Instead of having the last overloaded constructor, can you have a `val origin` in the companion object?